### DISCUSSION TITLE
Provide RustExplorer playground in issue template

### DISCUSSION THREAD

**Discussion by ChristianMurphy** [wooorm/markdown-rs#110] [Ideas]
Issues can be difficult to reproduce.
Having a sandbox provides a controlled and consistent environment for replicating an issue.
Potential starting point:

https://www.rustexplorer.com/b#LyoKW2RlcGVuZGVuY2llc10KbWFya2Rvd249IjEuMC4wLWFscGhhLjE2IgoqLwoKZm4gbWFpbigpIHsKICAgIHByaW50bG4hKCJ7fSIsIG1hcmtkb3duOjp0b19odG1sKCIjIyBIZWxsbywgKndvcmxkKiEiKSk7Cn0K

**Comment by wooorm** [comment]
Sure, very good idea!

Also, could be in the docs?
